### ISSUE TITLE
Doesn't work on Django 1.10

### ISSUE THREAD

**Issue by Zam4uK** [jorgecarleitao/django-sphinxql#8]
At least I faced with problem that in Django 1.10 removed add_aggregate func, I tryed replace by func add_annotation according to docs 1.8, but it raised another errror.Roll back to Django 1.9(

**Comment by robbietjuh** [comment]
According to Django 1.9 [code](https://github.com/django/django/blob/stable/1.9.x/django/db/models/sql/query.py#L976):

```
    def add_aggregate(self, aggregate, model, alias, is_summary):
        self.add_annotation(aggregate, alias, is_summary)
```

You should be good by just leaving out the `model` attribute.

**Comment by jorgecarleitao** [comment]
Commit 3f42f51 adds support for Django 1.10. The code with this commit was released as version 1.1.0 and it is now available in pypi: we can now install with `pip install django-sphinxql`. Thanks for pointing this out and proposing a solution, @Zam4uK and @robbietjuh!
